### DISCUSSION TITLE
YouTube Playlist download not working.

### DISCUSSION THREAD

**Discussion by AUTUMNIIIWIND** [subhra74/xdm#900] [Q&A]
Hi,
Thank you for creating XDM, it is for months my primary tool to download YouTube tutorials.
But I can't find a way to make YouTube Playlist download possible.

I tried: File / Download Video

But after searching, I do not see any results, just clean window.

There is an option for "advanced settings" with User Name and Password.
But I think it is only needed when you are trying to download from a website where access with User Name and Password is needed.

Anyway I need your help :)

Thank you

**Comment by SamirKhan21860** [comment]
Please solve the above mentioned issue because this is one of the best feature which I'm using since 2018. And this feature worth it. Please solve the problem as soon as possible. (File / Download Video)

**Comment by halilnevzat** [comment]
youtube playlist feature would be so great :( if its hard, im okay with download batch selected videos from browser extension. it catches videos but i have download one by one. so if there would be a batch selection in browser extension, it would also be great. thanks for your great work and time.

**Comment by subhra74** [comment]
Please try latest version 8.0.18, it supports playlist video
https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.18-beta
